widgets/contact/styles/default.less

```diff
@@ -1,4 +1,4 @@
-@import "../../../base/less/mixins";
```

Comment:
Ah, wasn't sure if you wanted to leave that as is or not - I'll make sure to revert it in the follow up commit.